thoth/common/openshift.py

```diff
@@ -143,6 +144,15 @@ def __init__(
                 "TLS verification when communicating with k8s/okd master is disabled"
             )
 
+        self.use_argo = bool(
+            int(os.getenv("THOTH_USE_ARGO", 0)) and use_argo
```

Comment:
This means both will need to be set - environment variable and the parameter passed in the constructor in order to run argo workflows. It would be better to control this purely on the environment variable or let the parameter override the value:

```python
    self.use_argo = use_argo or int(os.getenv("THOTH_USE_ARGO", 0))

Reply:
Updated, thanks :)